txpool/queue_priced.go

```diff
@@ -12,19 +11,16 @@ type pricedQueue struct {
 	queue *maxPriceQueue
 }
 
-func newPricedQueue() *pricedQueue {
-	q := pricedQueue{
-		queue: &maxPriceQueue{},
+// init initializes the underlying queue with initial transactions
+func (q *pricedQueue) init(baseFee uint64, initialTxs []*types.Transaction) *pricedQueue {
```

Comment:
This pattern... better to convert this to a `newPQ` function, don't you think?

Reply:
fixed https://github.com/0xPolygon/polygon-edge/pull/1696/commits/a9c2c519b9c25b0b36613bfb3802810172b957bb
init is removed, on each `Prepare` `newPricedQueue` is created